### ISSUE TITLE
[Collections/split] `.every` not working properly with Literal values?

### ISSUE THREAD

**Issue by RickBarretto** [arturo-lang/arturo#885]
**Describe the bug**
When I use `split.every` with literals, I get a flatten list.

**To Reproduce**
Running this code:
```
var: [
  'hello 'world 'from
  'arturo 'language 'have
  'a 'nice 'day
]

split.every: 3 'var
print var
; hello world from arturo language have a nice day
```

**Expected behavior**
The same of non-literals
```
var: [
  'hello 'world 'from
  'arturo 'language 'have
  'a 'nice 'day
]

print split.every: 3 var
; [hello world from] [arturo language have] [a nice day]
```

**Screenshots**
![image](https://user-images.githubusercontent.com/78623871/208486887-fcc53d55-2135-452e-b541-c1ec2c2bb62f.png)

**Desktop (please complete the following information):**
 - OS: amd64/windows
 - Version: v/0.9.80 b/3311

**Additional context**
from issue #871

**Comment by RickBarretto** [comment]
Also, I forgot to say. I'm just working on this issue! :wink:

**Comment by RickBarretto** [comment]
Alright, I discovered another issue. This time not related with literals, but with strings.
When I run, this:
```
    debug split.every: 3 "Manchester"
    debug split.every: 3 "Manchesterr"
```
I get this:
![image](https://user-images.githubusercontent.com/78623871/208494074-fedbe2b4-77e9-4df4-9e6e-5fce3b1c63e1.png)

It's filling this block with some strange character.
(Also, I redirected it from bash, so it's not a redirection/conversion error)
